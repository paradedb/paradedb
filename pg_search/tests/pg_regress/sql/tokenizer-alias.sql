DROP TABLE IF EXISTS need_alias;
CREATE TABLE need_alias(
    id serial8 not null primary key,
    title text,
    description text
);

INSERT INTO need_alias (title, description) VALUES ('the title', 'the description');

CREATE INDEX idxneed_alias ON need_alias USING bm25 (id, ((title || ' ' || description)::pdb.simple)) WITH (key_field = 'id');

SELECT * FROM paradedb.schema('idxneed_alias') ORDER BY name;