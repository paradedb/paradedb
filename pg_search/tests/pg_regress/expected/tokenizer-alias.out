DROP TABLE IF EXISTS need_alias;
CREATE TABLE need_alias(
    id serial8 not null primary key,
    title text,
    description text
);
INSERT INTO need_alias (title, description) VALUES ('the title', 'the description');
CREATE INDEX idxneed_alias ON need_alias USING bm25 (id, ((title || ' ' || description)::pdb.simple)) WITH (key_field = 'id');
ERROR:  indexed expression requires a tokenizer cast with an alias: (((title || ' '::text) || description))::pdb.simple
SELECT * FROM paradedb.schema('idxneed_alias') ORDER BY name;
ERROR:  relation "idxneed_alias" does not exist at character 31
